pages/team.rst

```diff
@@ -4,7 +4,8 @@
 .. tags: 
 .. category: 
 .. link: 
-.. description: Team
+.. subtitle: Team
```

Comment:
The description was used as the subtitle, adding a "subtitle" tag here makes it explicit.